src/System.Runtime/tests/System/Reflection/ReflectionTypeLoadExceptionTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+
+namespace System.Reflection.Tests
+{
+    public static class ReflectionTypeLoadExceptionTests
+    {
+        [Fact]
+        public static void NullExceptionsNoNullPointerException()
+        {
+            try
+            {
+                Type[] typo = new Type[1];
+                Exception[] excepto = new Exception[1];
+                throw new ReflectionTypeLoadException(typo, excepto, "Null elements in Exceptions array");
+            }
+            catch (ReflectionTypeLoadException e)
```

Comment:
You do need to actually throw and catch the exception for this validation. You can just create the exception without throwing it. E.g. it is what AggregateException tests do: https://github.com/dotnet/corefx/blob/master/src/System.Threading.Tasks/tests/AggregateExceptionTests.cs#L17